<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="name_label">
       <property name="text">
        <string>FIRST SCREEN</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name_line_edit"/>
     </item>
     <item>
      <widget class="QPushButton" name="change_name_button">
       <property name="text">
        <string>Change Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="go_to_second_button">
       <property name="text">
        <string>Go to 2nd Screen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exit_button">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
